Source/Utility/Utilities/UnitConversions.H

```diff
@@ -0,0 +1,243 @@
+#ifndef UNITCONVERSIONS_H
+#define UNITCONVERSIONS_H
+
+#include <AMReX.H>
+#include <AMReX_REAL.H>
+#include <AMReX_Utility.H>
+
+namespace pele::physics::utilities {
+
+// CGS to MKS conversions
+struct cgs2mks
+{
+  AMREX_GPU_HOST_DEVICE
+  AMREX_FORCE_INLINE
+  amrex::Real Length(const amrex::Real L_cgs)
+  {
+    return L_cgs * 1.0e-2; // cm to m
+  }
+
+  AMREX_GPU_HOST_DEVICE
+  AMREX_FORCE_INLINE
+  amrex::Real Length_Pow(const amrex::Real L_cgs, const amrex::Real n)
```

Comment:
maybe this is just my recent obsession with templating things, but I think this could be more efficient by having `n` be a template parameter rather than an argument, as then you'd be able to compute the `pow` in a `constexpr` rather than on every call at runtime.

Reply:
I love that ;)

Reply:
> I think this could be more efficient by having `n` be a template parameter rather than an argument, as then you'd be able to compute the `pow` in a `constexpr` at compile time rather than on every call at runtime.

I got rid of the Length_Pow function and edited the Length function:
```
template <int n = 0>
AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE amrex::Real
Length(const amrex::Real L_cgs)
{
  // Usage: Length(L_cgs) or Length<n>(L_cgs)
  if constexpr (n == 0) {
    return L_cgs * 1.0e-2; // cm to m
  } else {
    return L_cgs * std::pow(1.0e-2, n); // cm^n to m^n
  }
}
```

Also, I got rid of the `struct` and created namespaces as @marchdf suggested.

Reply:
Hmm I dug into this a bit and it looks like what I had in mind is not possible because `std::pow` is not constexpr capable. Also prior to C++20 you can't template on floats so this doesn't allow fractional exponents. I'd say go back to your original implementation, but instead of calling the power version `Length_Pow`, just have it overload `Length`. Sorry for misleading you.

Reply:
No worries.  I didn't know you couldn't template on floats until I tried, so this was a good exercise at least.  I'll push the updated version through soon.